### ISSUE TITLE
Migrate docs push jobs to Zuul

### ISSUE THREAD

**Issue by berendt** [osism/issues#219]
The builds of the documentations are already done by Zuul. The push not yet. This should be changed to Zuul. Afterwards, the Github Actions can be removed.

**Comment by osfrickler** [comment]
Depends-On: https://github.com/osism/issues/issues/226

**Comment by berendt** [comment]
* [ ] osism/documentation

**Comment by osfrickler** [comment]
Publishing the main documentation via zuul is still a todo item, but I wonder whether with the switch to Docusaurus this will be obsolete?

**Comment by berendt** [comment]
That's fine for the moment and will be resolved by the docusaurus migration.
